istioctl/cmd/deprecated_cmd.go

```diff
@@ -276,9 +276,8 @@ var (
 	}
 
 	getCmd = &cobra.Command{
-		Use:        "get <type> [<name>]",
-		Deprecated: "Use `kubectl get` instead (see https://kubernetes.io/docs/tasks/tools/install-kubectl)",
-		Short:      "Retrieve policies and rules",
+		Use:   "get <type> [<name>]",
```

Comment:
if you wish to revert this deprecation, can you identify if this was only reverted in master, or was reverted in a release branch?

Also I think this file is the wrong place to put it.